src/GeniusYield/Types/Credential.hs

```diff
@@ -84,26 +90,31 @@ paymentCredentialToBech32 (GYPaymentCredentialByScript sh) = serialiseToBech32Wi
 -- | Stake credential.
 data GYStakeCredential
        = GYStakeCredentialByKey !GYStakeKeyHash
-       | GYStakeCredentialByScript !GYValidatorHash
+       | GYStakeCredentialByScript !GYStakeValidatorHash
```

Comment:
Though this is a breaking change but as `GYStakeCredentialByScript` wasn't being made use of by any external API provided by Atlas, I assume it won't have effect for end users.